lab-david/__test__/photo-router.test.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+const request = require('superagent');
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'), { suffix: 'Prom'});
```

Comment:
Consider looking into fs-extra. Gives you promises natively and other useful methods.